.github/ISSUE_TEMPLATE/feedback_issue.yml

```diff
@@ -1,15 +1,15 @@
 name: Feedback Issue
-description: Open a feedback issue to gather feedback, suggestions, and experiences from users.
+description: Feedback issues are used to gather feedback, suggestions, and experiences from users while iterating on a specific product feature.
```

Comment:
Other descriptions seem to start with an action verb: 
<img width="589" alt="image" src="https://user-images.githubusercontent.com/666176/178219401-8ca83dce-85f4-4cea-a5d2-d2da3b831b4c.png">

Reply:
Good point, @atduarte! This exception could potentially help with holding back users from using the template for general feedback or bug reports, no?

Re-posting from the [relevant discussion](https://gitpod.slack.com/archives/C01KLC5750D/p1654877928794039?thread_ts=1652903673.228329&cid=C01KLC5750D) (internal) for visibility:

> ... to keep the flexibility we have with the autonomous product engineering teams and also allow room for new ideas[[1](https://github.com/gitpod-io/gitpod/issues/5712)][[2](https://github.com/gitpod-io/gitpod/issues/6576)][[3](https://github.com/gitpod-io/gitpod/issues/7603)][…], I’d not impose a specific body structure but just the label and possibly a title prefix for better discoverability.

>  ... ideally we could document this global practice as a product process so that everyone can read about this, understand the benefits, and help everyone practice this. For example, define what to include, when to open one, when to close one, and much more.

If still more users start pick up this template for submitting general feedback, I'd remove the issue template altogether.